src/latex/conway-ledger.tex

```diff
@@ -7,9 +7,35 @@
 \BODY
 }
 
-\include{preamble}
+\input{preamble}
 
 \begin{document}
+\title{Formal Specification of the Cardano Ledger for the Conway era}
+
+\author{
+   Andre Knispel \\ {\small \texttt{andre.knispel@iohk.io}} \\
+   William DeMeo \\ {\small \texttt{william.demeo@iohk.io}} \\
+   Joosep Jääger \\ {\small \texttt{joosep.jaager@iohk.io}} \\
+   }
```

Comment:
I realize this is not a math journal article (where we usually use alphabetical ordering when listing authors), but what's the motivation for using non-alphabetical ordering here?  Is the convention "project lead is first author" or something?